toplevel/g_toplevel.mlg

```diff
@@ -19,6 +19,7 @@ type vernac_toplevel =
   | VernacDrop
   | VernacQuit
   | VernacControl of vernac_control
+  | VernacShowGoal of int * int
```

Comment:
You can use inline records if you want `VernacShowGoal of { evar : int; sid: int }`